superset-frontend/package-lock.json

```diff
@@ -7670,28 +7670,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": "",
+              "resolved": false,
```

Comment:
Empty strings become `false`s again. 🤦‍♂

Reply:
lol this i can deal with :P